src/components/Toolbelt/Toolbelt.test.js

```diff
@@ -1,39 +1,53 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import { render, screen } from '@testing-library/react'
 
 import { fieldMode } from '../../enums'
 
 import { Toolbelt } from './Toolbelt'
 
-let component
-
-beforeEach(() => {
-  component = shallow(
-    <Toolbelt
-      {...{
-        fieldMode: fieldMode.OBSERVE,
-        handleFieldModeSelect: () => {},
-      }}
-    />
-  )
-})
-
-test('renders shop inventory', () => {
-  expect(component).toHaveLength(1)
-})
+describe('<ToolBelt />', () => {
+  const getSelectedButton = () => {
+    return screen
+      .getAllByRole('button')
+      .find(b => b.classList.contains('selected'))
+  }
 
-describe('tool selection', () => {
-  test('selects no tools by default', () => {
-    expect(component.find('.selected')).toHaveLength(0)
+  test('renders a button for each tool', () => {
+    render(<Toolbelt fieldMode={fieldMode.OBSERVE} />)
+    expect(screen.getAllByRole('button')).toHaveLength(3)
   })
 
-  describe('a tool is selected', () => {
-    beforeEach(() => {
-      component.setProps({ fieldMode: fieldMode.WATER })
+  describe('tool selection', () => {
+    test('there are no selected tools by default', () => {
+      render(<Toolbelt fieldMode={fieldMode.OBSERVE} />)
+      expect(getSelectedButton()).toBeUndefined()
+    })
+
+    test('marks the watering can selected for field mode WATER', async () => {
+      render(<Toolbelt fieldMode={fieldMode.WATER} />)
+      const button = screen.getByText('select watering can')
```

Comment:
this variable name ended up being confusing, i will probably rename it in the morning